docs/content/getting-started/quickstarts/quickstart-aws-s3/snippets/s3.bicep

```
@@ -0,0 +1,11 @@
+import aws as aws
+
+@description('The name of your S3 bucket.The AWS S3 Bucket name must follow the [following naming conventions](https://docs.aws.amazon.com/AmazonS3/latest/userguide/bucketnamingrules.html).')
```

Bicep descriptions don't follow markdown conventions

```suggestion
@description('The name of your S3 bucket.The AWS S3 Bucket name must follow the naming conventions described at https://docs.aws.amazon.com/AmazonS3/latest/userguide/bucketnamingrules.html')
```